### PR TITLE
sentry integration

### PR DESCRIPTION
**Pull request by trowik**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,12 +1,4 @@
-*.pyc
-*.swo
-*.swp
-.python-version
+*.**
```
Comment (winged) [inline .gitignore:1]: That's probably not a good idea. Why did you remove the ignore entries?

**Review by winged — CHANGES_REQUESTED** [review]
Looks awesome! However please revert the change to `.gitignore`, this would ignore every (new) file in the repo, which isn't what we want

File: .gitignore (hunk 1 above)
Comment (trowik) [reply .gitignore:1]: oh sorry. I was just testing. Didn't mean to commit it.

File: requirements-dev.txt [hunk 2]
```diff
@@ -4,4 +4,5 @@ ipdb==0.13.2
 flake8==3.7.9
 pytest==5.4.1
 pytest-django==3.9.0
-pytest-cov==2.8.1
\ No newline at end of file
+pytest-cov==2.8.1
+sentry-sdk==0.14.4
```
Comment (winged) [inline requirements-dev.txt:8]: sentry-sdk should be in the prod requirements, not dev

**Review by winged — APPROVED** [review]
